### PR TITLE
Change "issued identifiers list" to "issued identifiers map".

### LINKED ISSUE #36: Exact nature of the issued identifier list (editorial)
In [§4.4 Issue Identifier algorithm](https://w3c.github.io/rdf-canon/spec/#bn-issuer-state) the exact nature of _**issued identifiers list**_ is not really clear. Would not be clearer to say:

> List of mappings from existing identifiers to issued identifiers, in the order in which the issued identifiers were issued. This list is used to track the prevent issuing more than one new identifier per existing identifier, and to allow blank nodes to be reassigned identifiers some time after issuance.

Maybe even better to formalize these in terms of infra:

> a _list_ of _tuples_ of the form _(existing, issued)_. 

In general, I believe using infra for the description/definition of data structures might be better, it would avoid ambiguities.

B.t.w., is it really important to have it "in the order in which they were issued"? If that constraint wasn't there, a map could be used instead of tuples. So far, I did not see any point in the algorithm that relied on the order.

(Caveat: I only implemented the first degree hashes only. Maybe that is the reason.

### PR DESCRIPTION
**Pull request by gkellogg**
Fixes #36.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/39.html" title="Last updated on Nov 23, 2022, 6:13 PM UTC (35fc608)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/39/3b9dd9f...35fc608.html" title="Last updated on Nov 23, 2022, 6:13 PM UTC (35fc608)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -443,11 +443,10 @@ <h2>Blank Node Identifier Issuer State</h2>
       <dd>A counter that is appended to the <a>identifier prefix</a> to
         create an <a>blank node identifier</a>. It is initialized to
         <code>0</code>.</dd>
-      <dt><dfn>issued identifiers list</dfn></dt>
-      <dd>A list that tracks previously issued identifiers in the order in
-        which they were issued. It also tracks the existing identifier that
-        triggered the issuance, to prevent issuing more than one new identifier
-        per existing identifier, and to allow <a>blank nodes</a> to
+      <dt><dfn>issued identifiers map</dfn></dt>
+      <dd>A <a data-cite="INFRA#ordered-map">map</a> that relates existing identifiers to issued identifiers,
```
Comment (gkellogg) [inline spec/index.html:447]:
```suggestion
      <dd>An <a data-cite="INFRA#ordered-map">ordered map</a> that relates existing identifiers to issued identifiers,
```

File: spec/index.html [hunk 2]
```diff
@@ -657,14 +656,16 @@ <h3>Algorithm</h3>
         <var>issued identifier</var>. The steps of the algorithm are:</p>
 
       <ol>
-        <li id="iia-1">If there is already an issued identifier for
-          <var>existing identifier</var> in <a>issued identifiers list</a> of <var>I</var>,
+        <li id="iia-1">If there is a
+          <a data-cite="INFRA#map-entryå">map entry</a> for <var>existing identifier</var> in
```
Comment (dlongley) [inline spec/index.html:660]:
```suggestion
          <a data-cite="INFRA#map-entry">map entry</a> for <var>existing identifier</var> in
```

**Review by dlongley — APPROVED** [review]
Thanks! I didn't look closely to see if we missed any other occurrences. See my suggestion for a typo fix.

File: spec/index.html [hunk 3]
```diff
@@ -657,14 +656,16 @@ <h3>Algorithm</h3>
...
+        <li id="iia-1">If there is a
+          <a data-cite="INFRA#map-entry">map entry</a> for <var>existing identifier</var> in
+          <a>issued identifiers map</a> of <var>I</var>,
           return it.</li>
         <li id="iia-2">Generate <var>issued identifier</var> by concatenating
           <a>identifier prefix</a> with the string value of
           <a>identifier counter</a>.</li>
-        <li id="iia-3">Append an item to <a>issued identifiers list</a> that maps
-          <var>existing identifier</var> to <var>issued identifier</var>.</li>
+        <li id="iia-3">Add a <a data-cite="INFRA#map-entry">entry</a>
```
Comment (yamdan) [inline spec/index.html:666]:
```suggestion
        <li id="iia-3">Add an <a data-cite="INFRA#map-entry">entry</a>
```

**Review by yamdan — CHANGES_REQUESTED** [review]
Thank you @gkellogg, I added only small suggestion for a typo fix.

**Pull request merged**